packages/framework/src/Model/Administrator/Security/AdministratorFrontSecurityFacade.php

```diff
@@ -60,7 +60,12 @@ public function isAdministratorLogged()
     {
         try {
             $token = $this->getAdministratorToken();
-        } catch (\Shopsys\FrameworkBundle\Model\Administrator\Security\Exception\InvalidTokenException $e) {
+        } catch (
+            \Shopsys\FrameworkBundle\Model\Administrator\Security\Exception\InvalidTokenException |
+            \Symfony\Component\Security\Core\Exception\UnsupportedUserException |
+            \Symfony\Component\Security\Core\Exception\AuthenticationExpiredException |
+            \Symfony\Component\Security\Core\Exception\UsernameNotFoundException $e
```

Comment:
These exception are all extended from `Symfony\Component\Security\Core\Exception\AuthenticationException`. Even in this context it should be rater catch by the parrent one to prevent any other similar unhadled states.

Reply:
Good idea, updated.